src/main/java/com/github/dockerjava/api/model/Container.java

```diff
@@ -30,11 +34,17 @@
     @JsonProperty("Image")
     private String image;
 
+    /**
+     * @since since {@link RemoteApiVersion#VERSION_1_21}
+     */
+    @JsonProperty("ImageID")
+    private String imageId;
+
     @JsonProperty("Names")
     private String[] names;
 
     @JsonProperty("Ports")
-    public Port[] ports;
+    public ContainerPort[] ports;
```

Comment:
What about using a base class in a sub-package that is inherited by the nested class? 
So having a `com.github.docker-java.api.model.container.Port` that the nested class `com.github.docker-java.api.model.Container.Port` is inherited from? WDYT? To complicated?